### PR TITLE
issue #3 - remove privileged code from demo study

### PR DESCRIPTION
**Pull request by rhelmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,15 @@
 A demo add-on for Ion Studies.
 
-This requires [Firefox](https://www.mozilla.org/en-US/firefox/) 78.0 or newer, which supports the [new `browser.telemetry.submitEncryptedPing` WebExtension API](https://bugzilla.mozilla.org/show_bug.cgi?id=1634557).
+This is a normal, unprivileged WebExtension.
```
Comment (Dexterp37) [inline README.md:3]:
```suggestion
This is an unprivileged WebExtension.
```

File: background.js [hunk 2]
```diff
@@ -1,4 +1,15 @@
-/* eslint-disable no-undef */
+const ION_SIGNUP_URL = "https://mozilla-ion.github.io/ion-core-addon/";
```
Comment (Dexterp37) [inline background.js:1]: nit: this is missing a license header

File: background.js [hunk 3]
```diff
@@ -1,4 +1,15 @@
...
+const CORE_ADDON_ID = "ion-core-addon@mozilla.org";
```
Comment (Dexterp37) [inline background.js:2]:
```suggestion
const CORE_ADDON_FIREFOX_ID = "ion-core-addon@mozilla.org";
```

Note that this addon id is only valid in Firefox: Chromium-based browser requires to use certificate and self-signing (or proper signing from the extension store) to have a stable id. At least, that's what I understood!

**Review by Dexterp37 — APPROVED** [review]

File: background.js (hunk 3 above)
Comment (rhelmer) [reply background.js:2]: Yep, that's right... I've been waiting to cross that bridge until we come to it, I am thinking we should be careful in how we submit it so it doesn't get blocked or anything.

File: manifest.json [hunk 4]
```diff
@@ -1,35 +1,19 @@
...
-    "public_key": {
-      "id": "debug",
-      "key": {
-        "crv": "P-256",
-        "kty": "EC",
-        "x": "XLkI3NaY3-AF2nRMspC63BT1u0Y3moXYSfss7VuQ0mk",
-        "y": "SB0KnIW-pqk85OIEYZenoNkEyOOp5GeWQhS1KeRtEUE"
-      }
-    }
-  },
```
[the remark this answers is not in the document]
Comment (rhelmer) [reply manifest.json:32]: Good point! I guess we could pass it in a message, we'll have to figure out how to keep the overhead low.

[the remark this answers is not in the document]
Comment (rhelmer) [reply manifest.json:32]: Filed https://github.com/mozilla-ion/ion-core-addon/issues/47 for this and the "unimplemented" exception FIXME.

**Pull request merged**